tensorflow_io/core/kernels/kafka_kernels_deprecated.cc

```diff
@@ -25,6 +25,7 @@ limitations under the License.
 #include "tensorflow/core/framework/dataset.h"
 #include "tensorflow/core/framework/resource_mgr.h"
 #include "tensorflow/core/framework/resource_op_kernel.h"
+#include "tensorflow/core/lib/strings/stringprintf.h"
```

Comment:
@777ki this import is not needed now I guess?

Reply:
a oh, that is a mistake, i will remove it